R/runner.R

```diff
@@ -0,0 +1,59 @@
+runner_run <- function(orderly_root, reportname, parameters, branch, ref, ...) {
+  # Helper functions
```

Comment:
These are here because there is a bug in `rrq_task_create_call` and it doesn't seem to capture them. Made a ticket about this: https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-5153/Investigate-rrqtaskcreatecall-bug

Reply:
Does this work if you call `rrq::rrq_task_create_call` with the package qualifier? i.e. as `orderly.runner:::runner_run`?

Reply:
ohhh nevermind, i didnt install it, it did work!

Reply:
it actually works without it too, i just didnt do an install all and so it didnt pick it up from the `orderly.runner` package :sob: i was avoiding that cos I was relying on specific branches of the orderly2 and rrq repos and didnt know if install all would overwrite them with the head of the github repo, but yh that was the issue!